src/frontend/js/utils/api/joanie.ts

```diff
@@ -211,19 +218,15 @@ const API = (): Joanie.API => {
             method: 'POST',
             body: JSON.stringify(payload),
           }).then(checkStatus),
-        get: async (arg1) => {
+        get: async (filters) => {
```

Comment:
Nice, I'm going to re-use the way you refactored this function in the order PR too :)